deps.sh

```diff
@@ -5,13 +5,26 @@
 set -e
 
 # Test dependencies:
-tarantoolctl rocks install luatest 0.5.7
-tarantoolctl rocks install luacheck 0.25.0
-tarantoolctl rocks install luacov 0.13.0
-tarantoolctl rocks install luafilesystem 1.7.0-2
+tt rocks install luatest
```

Comment:
We could specify a `luatest` version here too.
```suggestion
tt rocks install luatest
```

Reply:
Both luatest 1.0.1 and 1.0.1-1 does not contain treegen, which is used in one of the tests.

Reply:
Please, add a comment like that:

```
Could be replaced with luatest >= 1.1.0 after a release.
```

It will help us do not forget the reason.